velox/exec/OutputBuffer.h

```diff
@@ -114,6 +144,9 @@ class DestinationBuffer {
   // the callback.
   DataAvailable getAndClearNotify();
 
+  // Returns the stats of this buffer.
+  BufferStats getStats() const;
```

Comment:
remove 'get' prefix

DestinationBufferStats stats() const;

Consider moving DestinationBufferStats into DestinationBuffer class: DestinationBuffer::Stats

Reply:
`get` prefix removed, but after some trying I tend to leave DestinationBufferStats outside of DestinationBuffer because moving it inside would make the DestinationBuffer class harder to read.